### PR TITLE
Improve russian translation

### PR DESCRIPTION
**Pull request by KhraksMamtsov**
- [+] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [+] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [+] My pull request targets the `master` branch of freeCodeCamp.
- [+] None of my changes are plagiarized from another source without proper attribution.
- [+] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [+] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md [hunk 1]
```diff
@@ -3,26 +3,24 @@ id: 587d7fb1367417b2b2512bf4
 title: Chain Middleware to Create a Time Server
 challengeType: 2
 forumTopicId: 301510
-localeTitle: Промежуточное ПО для создания сервера времени
+localeTitle: Middleware для создания сервера времени
 ---
 
-## Description
+## Описание
```
Comment (greggubarev) [inline curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: @RandellDawson is it ok - translating such this line?

Comment (thecodingaviator) [reply curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: Nope. I think it should remain `Description` itself.

File: curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md [hunk 2]
```diff
@@ -3,26 +3,24 @@ id: 587d7fb1367417b2b2512bf4
...
-В маршруте <code>app.get('/now', ...)</code> функцию промежуточного программного обеспечения и конечный обработчик. В функции промежуточного программного обеспечения вы должны добавить текущее время к объекту запроса в ключе <code>req.time</code> . Вы можете использовать <code>new Date().toString()</code> . В обработчике ответьте объектом JSON, взяв структуру <code>{time: req.time}</code> . 
-Подсказка: тест не пройдет, если вы не подключите промежуточное ПО. Если вы смонтируете функцию где-то еще, тест не пройдёт, даже если результат вывода верный.
+Этот подход полезен для разделения операций сервера на более мелкие единицы. Это приводит к лучшей структуре приложения и возможности повторного использования кода в разных местах. Этот подход также можно использовать для проверки данных. В каждой точке стека middleware вы можете заблокировать выполнение текущей цепочки и передать управление функциям, специально разработанным для обработки ошибок. Или вы можете передать управление следующему подходящему маршруту для обработки особых случаев. Мы увидим, как в расширенном разделе express. 
 </section>
-
-## Instructions
+  
+## Задание
```
Comment (thecodingaviator) [inline curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: Revert this change

File: curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md (hunk 1 above)
Comment (thecodingaviator) [inline curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: Revert this change

File: curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md [hunk 3]
```diff
@@ -3,26 +3,24 @@ id: 587d7fb1367417b2b2512bf4
...
 <section id='instructions'>
-In the route <code>app.get('/now', ...)</code> chain a middleware function and the final handler. In the middleware function you should add the current time to the request object in the <code>req.time</code> key. You can use <code>new Date().toString()</code>. In the handler, respond with a JSON object, taking the structure <code>{time: req.time}</code>.
-<strong>Note:</strong> The test will not pass if you don’t chain the middleware. If you mount the function somewhere else, the test will fail, even if the output result is correct.
+В маршруте <code>app.get('/now', ...)</code> привяжите middleware к конечному обработчику. В middleware добавьте текущее время к объекту запроса в поле <code>req.time</code>. Вы можете использовать <code>new Date().toString()</code> . В обработчике ответьте объектом JSON со структурой <code>{time: req.time}</code> . 
+<strong>Подсказка:</strong> тест не пройдет, если вы не подключите middleware. Если вы смонтируете ee где-то еще, тест не пройдёт, даже если результат вывода верный.
 </section>
 
-## Tests
+## Тесты
```
Comment (thecodingaviator) [inline curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: Revert this change

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Closing as stale/decayed.

Hi,

Thanks a lot for your pull request. We have not seen an update from you on this PR in a while. We are going to have to close this as stale. If you are still interested, please open a fresh PR.

In case you believe this is closed incorrectly please hit us up in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors) to get this re-opened.

Thanks and happy contributing.
